libs/react-mapbox/src/components/Cluster/Cluster.tsx

```diff
@@ -40,6 +40,30 @@ export const Cluster = ({
     [features],
   );
 
+  const adjustCoordinates = () => {
+    const updatedFeatures: Array<Supercluster.PointFeature<Supercluster.AnyProps>> = [];
+
+    pointFeatures.forEach((f, index) => {
+      const sameIndexPoint = updatedFeatures.findIndex(pf => pf.geometry.coordinates[0] === f.geometry.coordinates[0] && pf.geometry.coordinates[1] === f.geometry.coordinates[1]);
+
+      let offset = 0.0001;
+
+      if (sameIndexPoint > -1 && sameIndexPoint !== index) {
+        const updatedFeature = { ...f };
+        updatedFeature.geometry.coordinates[0] = updatedFeature.geometry.coordinates[0] - offset;
+        offset += 0.0001;
+
+        updatedFeatures.push(updatedFeature);
+      }
+      else updatedFeatures.push(f);
+    })
+
+    return updatedFeatures;
+  }
```

Comment:
ak budu viac ako 2 body na novom mieste tymto si ich velmi lahko vies naposuvat na nove, ale stare rovnake miesto. Robit to cele v jednom cykle si trochu koleduje o problemy tohto druhu.

Ak mas este inu pracu na stacku tak toto by som mozno nechal nateraz otvorene, beruc do uvahy ze sa nam kratia dostupne hodiny.